README.md

```diff
@@ -1,18 +1,30 @@
 Bind container image
 ====================
 
-This repository contains Dockerfile for [BIND](https://www.isc.org/downloads/bind/) based on [baseruntime](""https://hub.docker.com/r/baseruntime/baseruntime/) for the Fedora 26 Boltron general usage.
+This repository contains Dockerfile for [BIND](https://www.isc.org/downloads/bind/) based on [baseruntime](""https://hub.docker.com/r/baseruntime/baseruntime/).
 For more information about modules see official [Fedora Modularity documentation](docs.pagure.org/modularity/).
 
 
 Configuration
 ----------------------------------
 
-There are two options for running DNS server:
- - Authoritative
-    - you need set domains in [config.yaml](./files/authoritative-dns/config.yaml)
- - Caching (recursive)
-    - you can set nameservers in [config.yaml](./files/caching-dns/config.yaml)
+There are two options for running Bind DNS server:
+
+### 1) Authoritative
+
+Authoritative-only servers only respond to iterative queries for the zones that they are authoritative for. This means that if the server does not know the answer, it will just tell the client (usually some kind of resolving DNS server) that it does not know the answer and give a reference to a server that may know more.
```

Comment:
Good write-up. The important note is probably that zones usually represent your company.